server/routes.go

```diff
@@ -49,6 +49,7 @@ type getRoutesResponse struct {
 	ExternalLinks      getExternalLinksResponse           `json:"external"`         // All external links for the client to use
 	OrganizationConfig getOrganizationConfigLinksResponse `json:"orgConfig"`        // Location of the organization config endpoint
 	Flux               getFluxLinksResponse               `json:"flux"`
+	ValidTextTemplates string                             `json:"validTextTemplates"` // Location of the valid text templates endpoint
```

Comment:
```suggestion
	ValidateTextTemplates string                             `json:"validateTextTemplates"` // Location of the text template validation route
```

Reply:
I've got this change locally.  Pushing up now 😄